src/common/src/array/jsonb_array.rs

```diff
@@ -23,10 +24,10 @@ use crate::collection::estimate_size::EstimateSize;
 use crate::types::{Scalar, ScalarRef};
 use crate::util::iter_util::ZipEqFast;
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq, Display)]
 pub struct JsonbVal(Box<Value>); // The `Box` is just to keep `size_of::<ScalarImpl>` smaller.
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Display)]
```

Comment:
`Display` is so easy to be used implicitly (`.to_string`) without careful thoughts. We have our own `ToText` which is intended to convert a typed value to string in a PostgreSQL compatible way.

For the use case of interacting with a udf, both auto-derived and pg-compatible string would work. But I am concerned about having 2 versions and the non-compatible one being more easy to use.

Reply:
I always thought they were the same.
For convenient, what about implementing Display as the pg-compatible format?